modules/tinymce/src/core/test/ts/browser/FormatterCheckTest.ts

```diff
@@ -163,6 +163,63 @@ describe('browser.tinymce.core.FormatterCheckTest', () => {
     assert.isTrue(editor.formatter.match('complex', { color: '#ff00' }), 'Selected element match with variable and function');
   });
 
+  context('using similar', () => {
+    const setupRegister = (editor: Editor) => {
+      editor.formatter.register({
+        tablecellverticalalign: {
+          selector: 'td,th',
+          styles: {
+            'vertical-align': '%value'
+          },
+          remove_similar: true
+        }
+      });
+    };
+
+    const setContentWithoutAttribute = (editor: Editor) => {
+      editor.getBody().innerHTML = (
```

Comment:
Please do not use direct dom modification to set the initial content like this... please use the relevant `setContent` APIs instead.

- [ ] Don't use `editor.getBody().innerHTML`